### PR TITLE
loop over logs&traces flush handlers

### PR DESCRIPTION
**Pull request by schehata**

### CODE REVIEW

**Comment by schehata** [comment]
@bahlo still couldn't figure out the provider shutdown issue. maybe we can move forward with current status (tracing works) until we figure out a fix for the issue.

**Comment by schehata** [comment]
> @bahlo still couldn't figure out the provider shutdown issue. maybe we can move forward with current status (tracing works) until we figure out a fix for the issue.

arne nailed it, problem is fixed.

File: examples/prisma-axiom/index.ts [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import withAxiom from '../../src/axiom';
 import { PrismaClient } from '@prisma/client';
+const express = require('express')
```
Comment (bahlo) [inline examples/prisma-axiom/index.ts:3]: Can we split `index.ts` and `main.ts` into `script` and `express` example folders

File: examples/prisma-axiom/main.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  const user = await prisma.user.findFirst();
+  console.log('new user created', user);
+
+  await prisma.user.deleteMany();
+
+}
+
+main().finally(async () => {
+  console.log('disconnct prisma')
```
Comment (bahlo) [inline examples/prisma-axiom/main.ts:23]:
```suggestion
  console.log('disconnect prisma')
```

**Review by bahlo — APPROVED** [review]
nice nice nice

File: examples/prisma-axiom/index.ts (hunk 1 above)
Comment (schehata) [reply examples/prisma-axiom/index.ts]: done.

**Pull request merged**